crates/jose/src/jwk/public_parameters.rs

```diff
@@ -283,102 +232,58 @@ mod rsa_impls {
 
 mod ec_impls {
```

Comment:
I removed some From impls to avoid buggy conversions, especially with RSA signatures